Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1062,6 +1066,43 @@ protected virtual string CreateSafeChartName(string chartName)
             return Uri.EscapeDataString(chartName);
         }
 
+        /// <summary>
+        /// Samples benchmark continuously, and on date change, samples performance and portfolio equity
+        /// </summary>
+        /// <param name="time">Current time step of the AlgorithmManager loop</param>
+        /// <param name="force">Forces processing of equity and performance if true</param>
+        public virtual void Sample(DateTime time, bool force = false)
```

Comment:
Seems both live and backtesting implementations are very similar, can we join them and place it into the `BaseResultsHandler`? I think the difference is that Live is sampling the `Benchmark` on each call?

Reply:
Personally, I think it's best that each `Sample` method is implemented separately since they have their own concerns with how they're sampling. Mixing these concerns might make us reliant on specific base class behavior. 

I'll talk to you more about this, would like to know what your thoughts are 👍

Reply:
Update: we looked into addressing this review, but we concluded that it would require additional effort outside the scope of this PR.